### PR TITLE
Hotfix - Ignore `<template>` when calculating grid items

### PR DESCRIPTION
**Pull request by ThatIsAPseudo**
When using `x-masonry` on an element whose children items are generated through a `<template>` element, this `<template>` wasn't excluded from the `gridItems` array.  This led to alignment issues because of miscomputed margins.

(This change wasn't tested yet as I'm not familiar with the build process.)

### CODE REVIEW

File: src/useMasonry.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export function useMasonry(el) {
   const gridGap = parseFloat(getComputedStyle(el).gap)
   const gridItems = [...el.childNodes].filter(
-    (gridItem) => gridItem.nodeType === 1
+    (gridItem) => ((gridItem.nodeType === 1) and (gridItem.tagName != 'TEMPLATE'))
```
Comment (markmead) [inline src/useMasonry.js]: You can't write `and` in JavaScript, it would be `&&`.

Comment (markmead) [reply src/useMasonry.js]: It should be using `!==` for extra strictness.

**Review by markmead — CHANGES_REQUESTED** [review]
Happy to do the build stuff, just a few fixes.

File: src/useMasonry.js (hunk 1 above)
Comment (ThatIsAPseudo) [reply src/useMasonry.js]: Thanks for pointing these out, I should've seen that.

**Pull request merged**